src/collections/unordered-map.ts

```diff
@@ -107,15 +132,19 @@ export class UnorderedMap<DataType> {
   }
 
   // converting plain object to class object
-  static reconstruct<DataType>(data: UnorderedMap<DataType>): UnorderedMap<DataType> {
+  static reconstruct<DataType>(
+    data: UnorderedMap<DataType>
+  ): UnorderedMap<DataType> {
     // removing readonly modifier
     type MutableUnorderedMap = Mutable<UnorderedMap<DataType>>;
-    let map = new UnorderedMap(data.prefix) as MutableUnorderedMap;
+    const map = new UnorderedMap(data.prefix) as MutableUnorderedMap;
+
     // reconstruct keys Vector
-    map.keys = new Vector(data.prefix + "u");
+    map.keys = new Vector(`${data.prefix}u`);
```

Comment:
Is this better than `+`?

Reply:
@ailisp It is not better in any performance or conventions, I just prefer it to be template literals, we are using them elsewhere in code so thought might as well keep it the same all over.